Picture.py

```diff
@@ -35,7 +35,15 @@ def setTitle(self, name):
         self.title = name
 
     def getPixels(self):
-        return self.image.getdata()
+        colList = list(self.image.getdata())
+        pixList = list()
+        counter = 0
+        for i in colList:
+            x = counter%self.width
+            y = (counter-x)/self.width
+            pixList.append(Pixel(self.getImage,x,y))
+            counter+=1
+        return pixList
```

Comment:
We need to think more carefully about this.  Given that we're not going to store color values in the pixel object, we don't really need to get them using getdata().  Notice that the only thing colList is used for is to iterate over the pixels.  We need to think more carefully about this, but I think it's going to be okay to just create a list of pixel objects without actually needing to call getdata().